mne/minimum_norm/inverse.py

```diff
@@ -1454,8 +1471,8 @@ def _prepare_forward(forward, info, noise_cov, fixed, loose, rank, pca,
             convert_forward_solution(
                 forward, surf_ori=True, force_fixed=True,
                 use_cps=use_cps, copy=False)
-    else:
-        if loose.get('surface', 1.) < 1:
+    else:  # Free or loose orientation
+        if any(v < 1 for v in loose.values()):
```

Comment:
I like the simplification you made with computing and storing `fixed_inverse` once. But I see this `any` conditional show up twice -- can we make a similar simplification by computing it once toward the top as well?

Reply:
Good catch!